test/e2e/util.go

```diff
@@ -58,6 +58,13 @@ const (
 
 	// How often to poll pods.
 	podPoll = 5 * time.Second
+
+	// service accounts are provisioned after namespace creation
+	// a service account is required to support pod creation in a namespace as part of admission control
+	serviceAccountProvisionTimeout = 2 * time.Minute
```

Comment:
is this what you were seeing? I've never seen it take more than 10 seconds even under heavy load on an underpowered laptop...